orchagent/portsorch.cpp

```diff
@@ -1179,15 +1251,25 @@ bool PortsOrch::addBridgePort(Port &port)
         return false;
     }
 
+    m_portList[port.m_alias] = port;
     SWSS_LOG_NOTICE("Add bridge port %s to default 1Q bridge", port.m_alias.c_str());
 
     return true;
 }
 
-bool PortsOrch::removeBridgePort(Port port)
+bool PortsOrch::removeBridgePort(Port &port)
 {
     SWSS_LOG_ENTER();
 
+    if (port.m_vlan_members.size() > 0)
+    {
+        for (auto &vme: port.m_vlan_members)
+        {
+            SWSS_LOG_DEBUG("Port %s still in VLAN %d", port.m_alias.c_str(), vme.first);
+        }
+        return true;
```

Comment:
this should be false.

Reply:
each time  a port/lag leaves a VLAN,  " if (removeVlanMember(vlan, port) && removeBridgePort(port))"  is called,    port.m_vlan_members.erase(vlan_member) is called in removeVlanMember(vlan, port), and here it is to check whether the port/lag is still in any vlan.  It is ok to be non-empty.

Reply:
this make the return value confusing, according to the function name the return value true means you have removed the bridge port, in this case you should return false.